### ISSUE TITLE
Need to bump BSP submodule.

### ISSUE THREAD

**Issue by T-K-233** [ucb-ee290c/RoboChip-Bringup#1]
The robochip.h is not included in the current bsp directory.

**Comment by Andalite1999** [comment]
yeah, it's in bsp/robochip/inc/robochip.h

**Comment by T-K-233** [comment]
Are you using [bsp @ 22e5d70](https://github.com/ucb-bar/Baremetal-IDE-bsp/tree/22e5d709a01c53e64e9fdd890751af692f4d873d)? I think you have a more recent commit on your local end.

**Comment by Andalite1999** [comment]
No, I'm using this: https://github.com/ucb-ee290c/RoboChip-Bringup/commit/908c2c6

**Comment by T-K-233** [comment]
let's solve this during tomorrow's worksession
